docs/developer-docs/setup/cycles/cycles-ledger.md

```diff
@@ -0,0 +1,158 @@
+- Trading other tokens for ICP on a decentralized exchange.
+- Receiving tokens as rewards for participating in the governance of the Internet Computer.
+- Receiving a grant of tokens through the Internet Computer Association (ICA) or the DFINITY Foundation.
+- Receiving tokens as remuneration for providing computing capacity as a node provider.
+
+Once you have obtained ICP, you also need to have an account controlled via `dfx` to transfer those tokens into. This step is necessary to create cycles and canisters. To get an account ID for an account controlled via `dfx`, first determine which `dfx` identity you are using with the command:
+
+```
+dfx identity whoami
+```
+
+If you're using the default identity, you should create a new identity with the command:
+
+```
+dfx identity new IDENTITY_NAME
+```
+
+Replace `IDENTITY_NAME` with the an identity name of your choice, such as 'MyIdentity`.
+
+Your identity's seed phrase will be returned. Be sure to save this in a secure location. Then, set this identity to be used by the current session:
+
+```
+dfx identity use IDENTITY_NAME
+```
+
+Then, to get the account ID for this identity, use the command:
+
+```
+dfx ledger account-id
+```
+
+This will display your account number on the ICP ledger. It looks similar to this:
+
+```
+e213184a548871a47fb526f3cba24e2ee2fbbc8129c4ab497ef2ce535130a0a4
+```
+
+Then, to return the principal ID associated with your identity, run the command:
+
+```
+dfx identity get-principal
+```
+
+This principal ID will be where you need to send your ICP tokens to create cycles and canisters. It will also be used when others want to send you cycles using the cycles ledger. 
+
+### Checking your cycles balance
+
+To check the cycles balance of your user principal ID, use the command:
+
+```
+dfx cycles balance --network ic
+```
+
+### Converting ICP to cycles
+
+To convert ICP into cycles using the cycles ledger, use the command:
+
+```
+dfx cycles convert AMOUNT --network ic
+```
+
+Replace `AMOUNT` with the number of ICP to convert into cycles, such as `2.7`. Alternatively, the amount can be specified in terms of ICP and e8s (where `100,000,000 e8s = 1 ICP`) using the options `--icp` and `--e8s`. For example, `2.7` would be expressed as `--icp 2 --e8s 70_000_000`.
+
+### Transferring cycles to a principal
+
+To transfer cycles to another principal ID, use the command:
+
+```
+dfx cycles transfer AMOUNT PRINCIPAL_ID (--subaccount [SUBACCOUNT]) --network ic
+```
+
+Replace `AMOUNT` with the number of cycles, such as `34000000`, and `PRINCIPAL_ID` with the principal ID you'd like to transfer the cycles to, such as `tsqwz-udeik-5migd-ehrev-pvoqv-szx2g-akh5s-fkyqc-zy6q7-snav6-uqe`. Optionally, a subaccount can be provided, in which case the recipient will receive the funds in the provided subaccount.
```

Comment:
Maybe useful: it's finally possible to use suffixes like `500k` or `3TC`. See all options [here](https://github.com/dfinity/sdk/blob/master/CHANGELOG.md#feat-accept-more-ways-to-specify-cycle-and-e8s-amounts)

Reply:
Yes, I think it makes sense to add this information here (in particular when dealing with numbers in the trillions)!